.github/ISSUE_TEMPLATE/performance-challenge-program.md

```diff
@@ -1,21 +1,25 @@
 ---
-name: "✈️ Performance Challenge Program"
-about: PCP Season1 Program
-title: 'PCP: '
-labels: ''
-assignees: ''
+name: "✈️ Performance Challenge Program (PingCAP Internal)"
+title: 'PCP-X: '
 
 ---
 
-## PCP Issue
+## Description
 
-- difficult: medium
-- score: 150
-- min score required: 100
-- expired: 7d
-- Recommended Skills:
-- Mentor(s):
+To be filled
 
-## Description
+## Difficulty
+
+* To be filled
```

Comment:
can you explain detailedly which Difficulty should we write here? easy, medium or hard?